docs/ingest-api-walkthrough.md

```diff
@@ -66,7 +66,7 @@ Evaluate the expression `_links.protocols.href` from the response to the previou
 Any files in the DCP that were used as an analysis inputs should be linked to the analysis process. Given an input file UUID, this can be performed as follows:
 
 ```bash
->: curl -X POST -H "Content-Type: application/json" http://api.ingest.dev.data.humancellatlas.org/processes/{analysis_id}/add-input-file -d '{"inputFileReference": "<input file UUID>"}'
+>: curl -X POST -H "Content-Type: application/json" http://api.ingest.dev.data.humancellatlas.org/processes/{analysis_id}/inputFiles -d '{"inputFileReference": "<input file UUID>"}'
```

Comment:
Sorry, I just noticed now, inputFileReference  is a bit vague. What if we have `fileUuid` param, instead? Or `fileUuids` so that they would only need to make 1 request to link all files?

Reply:
I'll change it to inputFileUuid.

I don't want to accept a list here as it increases the chances of an optimistic lock exception.

Reply:
Just looked at the code, it already is inputFileUuid; the documentation is just wrong. Fixing now